### DISCUSSION TITLE
How do you specify within-group order of rows

### DISCUSSION THREAD

**Discussion by andtheWings** [rstudio/gt#1110] [Q&A] [answered]
I've been scanning the reference section and it seems like `row_group_order()` allows you to specify the order of the groups in relation to each other, but is there also a way to specify the order of rows within each group?

**Comment by rich-iannone** [comment, marked as answer]
You're correct with the first part and, no, unfortunately there is no way to specify the within-group row ordering. Would you mind submitting an issue asking for this feature?

**Reply by andtheWings** [reply]
Done!
